app.js

```diff
@@ -0,0 +1,79 @@
+const timeButton = document.querySelectorAll(".time-select button");
+const timeValue = document.querySelector(".time-display");
+const mudeButton = document.querySelectorAll(".sound-picker button");
+const playButton = document.querySelector(".play");
+const replayButton = document.querySelector(".replay");
+
+const video = document.querySelector("video");
+const sound = document.querySelector("audio");
+
+const outline = document.querySelector(".moving-outline circle");
+const outlineLength = outline.getTotalLength();
+outline.style.strokeDashoffset = outlineLength;
+outline.style.strokeDasharray = outlineLength;
+
+let fakeDuratation = 600;
+let minute = Math.floor(fakeDuratation / 60);
+let second = Math.floor(fakeDuratation % 60);
+timeValue.textContent = `${minute} : ${second}`;
+
+timeButton.forEach(button => {
+  button.addEventListener("click", function() {
+    fakeDuratation = this.getAttribute("data-time");
+    minute = Math.floor(fakeDuratation / 60);
+    second = Math.floor(fakeDuratation % 60);
+    timeValue.textContent = `${minute} : ${second}`;
+  })
+});
+
+mudeButton.forEach(button => {
+  button.addEventListener("click", function() {
+    video.src = this.getAttribute("data-video");
+    sound.src = this.getAttribute("data-sound");
+  })
+});
+
+playButton.addEventListener("click", playSondAndVideo);
+
+function playSondAndVideo() {
+  if(sound.paused) {
+    video.play();
+    sound.play();
+    playButton.src = "/svg/pause.svg";
```

Comment:
check all paths, there are some errors in the console

![image](https://user-images.githubusercontent.com/18407086/101295600-b66ac100-382f-11eb-8f8f-2989f8ea8c83.png)